### PR TITLE
Multiple issue fixed after QA testing

### PR DESCRIPTION
**Pull request by danishjamal104**
1. Introduced new application variable till.
2. Correlation id mapping fixed.
3. Zeeebe variable updated.
4. Callback body parsing fixed.

### CODE REVIEW

File: src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java [hunk 1]
```diff
@@ -72,6 +72,12 @@ public void configure() {
                 .setHeader(Exchange.HTTP_METHOD, constant("GET"))
                 .setHeader("Authorization", simple("Basic " + createAuthHeader(clientKey, clientSecret)))
                 .setHeader(Exchange.HTTP_RAW_QUERY, constant("grant_type=client_credentials"))
+                .process(exchange -> {
+                    logger.info("\nClient key: " + clientKey);
+                    logger.info("\nSecret key: " + clientSecret);
```
Comment (SubhamPramanik) [inline src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java:77]: Should we log these sensitive data?

File: src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java [hunk 2]
```diff
@@ -97,6 +103,8 @@ public void configure() {
     }
 
     private String createAuthHeader(String key, String secret) {
+        key = key.replace("\n", "");
+        secret = secret.replace("\n", "");
```
Comment (SubhamPramanik) [inline src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java:107]: Same here as well?

Comment (danishjamal104) [reply src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java:107]: here the data is only processed and not logged.

File: src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java (hunk 1 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/connector/mpesa/auth/AuthRoutes.java:77]: fixed

**Review by avikganguly01 — APPROVED** [review]

**Pull request merged**